tests/sentry/api/endpoints/test_auth_login.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+import six
+from django.core.urlresolvers import reverse
+from mock import patch
+from exam import fixture
+
+from sentry.testutils import APITestCase
+
+
+class AuthLoginEndpointTest(APITestCase):
+    @fixture
+    def path(self):
+        return reverse('sentry-api-0-auth-login')
+
+    def setUp(self):
+        # Requets to set the test cookie
```

Comment:
I don't think the new endpoint uses test cookies.

Reply:
It does. In your original branch you removed it from the form, but I added it back and just made sure the config endpoint sets the test cookie.